lib/px/pxnginx.lua

```diff
@@ -55,6 +55,10 @@ function M.application(file_name)
     local reverse_prefix = string.sub(px_config.px_appId, 3, string.len(px_config.px_appId))
     local lower_request_url = string.lower(ngx.var.request_uri)
 
+    local function is_first_party_proxy_request(reverse_prefix, lower_request_url)
+        return (px_client.reverse_px_client(reverse_prefix, lower_request_url) or px_client.reverse_px_xhr(reverse_prefix, lower_request_url) or px_client.reverse_px_captcha(reverse_prefix, lower_request_url))
```

Comment:
why not just `lower_request_url`s that start with `reverse_prefix`?

Reply:
already doing it inside the functions, should we do it again?